spec/lib/group_spec.rb

```diff
@@ -81,83 +81,217 @@
 
     let(:group) { ActAsGroup::Group.create(type: document_type, ids: document_ids, owner_id: owner_id) }
```

Comment:
I know that line wasn't modified by you, but do you mind breaking the line to have less than 100 characters? Thanks!